msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalSdk.java

```diff
@@ -145,6 +159,15 @@ public MsalAuthResult acquireTokenSilent(@NonNull MsalAuthTestParams authTestPar
             acquireTokenParametersBuilder.withClaims(authTestParams.getClaims());
         }
 
+        if (authTestParams.getAuthScheme() == AuthScheme.POP) {
+            acquireTokenParametersBuilder.withAuthenticationScheme(
+                    PoPAuthenticationScheme.builder()
+                            .withHttpMethod(HttpMethod.valueOf("GET"))
+                            .withUrl(new URL("https://signedhttprequest.azurewebsites.net/api/validateSHR"))
+                            .build()
+            );
+        }
```

Comment:
I see similar code above for interactive. It can probably move to a helper method in this class

`getAuthenticationScheme()`